examples/minimal-logs/main.tf

```diff
@@ -0,0 +1,156 @@
+  read_write_paths   = [""] # r/w policy permitting specified rw actions on entire bucket
+  additional_tags    = var.tags
+}
+
+# Create new EC2 key pair
+#resource "tls_private_key" "tamr_ec2_private_key" {
+#  algorithm = "RSA"
+#}
+
+module "aws-vm-sg-ports" {
+  #source = "git::https://github.com/Datatamer/terraform-aws-tamr-vm.git//modules/aws-security-groups?ref=2.0.0"
+  source = "../../modules/aws-security-groups"
+}
+
+module "aws-sg" {
+  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id = aws_vpc.tamr_vm_vpc.id
+  ingress_cidr_blocks = [
+    "10.0.0.0/24"
+  ]
+  egress_cidr_blocks = [
+    "0.0.0.0/0"
+  ]
+  ingress_ports    = module.aws-vm-sg-ports.ingress_ports
+  sg_name_prefix   = var.name-prefix
+  ingress_protocol = var.ingress_protocol
+  egress_protocol  = "all"
+  tags             = var.tags
+}
+
+module "tamr-vm" {
+  source                      = "git::git@github.com:Datatamer/terraform-aws-tamr-vm.git?ref=4.2.0"
+  aws_role_name               = format("%s-tamr-ec2-role", var.name-prefix)
+  aws_instance_profile_name   = format("%s-tamr-ec2-instance-profile", var.name-prefix)
+  aws_emr_creator_policy_name = format("%sEmrCreatorPolicy", var.name-prefix)
+  s3_policy_arns = [
+    module.s3-bucket.rw_policy_arn,
+  ]
+  ami               = var.ami_id
+  instance_type     = "r5.2xlarge"
+  key_name          = var.key_name
+  availability_zone = local.az
+  vpc_id            = aws_vpc.tamr_vm_vpc.id
+  subnet_id         = aws_subnet.tamr_vm_subnet.id
+  bootstrap_scripts = [
+
+    # NOTE: If you would like to use local scripts, you can use terraform's file() function
+    templatefile("./test-bootstrap-scripts/cloudwatch-install.sh", { region = data.aws_region.current.name, endpoint = module.endpoints.endpoints["logs"].dns_entry[0]["dns_name"] }),
+  ]
+
+  security_group_ids = module.aws-sg.security_group_ids
+  tags               = var.tags
+}
+
+resource "aws_route_table" "example" {
+  vpc_id = aws_vpc.tamr_vm_vpc.id
+  tags   = var.tags
+}
+
+resource "aws_route_table_association" "a" {
+  subnet_id      = aws_subnet.tamr_vm_subnet.id
+  route_table_id = aws_route_table.example.id
+}
+
+module "endpoints" {
+
+  source = "terraform-aws-modules/vpc/aws//modules/vpc-endpoints"
+
+  vpc_id = aws_vpc.tamr_vm_vpc.id
+
+  endpoints = {
+    s3 = {
+      service_type    = "Gateway"
+      service         = "s3"
+      tags            = { Name = format("%s-%s", var.name-prefix, "s3-vpc-endpoint") }
+      route_table_ids = flatten([aws_route_table.example.id])
+    },
+    logs = {
+      service_type        = "Interface"
+      service             = "logs"
+      tags                = { Name = "cloudwatch-vpc-endpoint" }
+      private_dns_enabled = true
+      security_group_ids  = [aws_security_group.interface_endpoint.id]
+      subnet_ids          = [aws_subnet.tamr_vm_subnet.id]
+    },
+  }
+  tags = var.tags
+}
+
+resource "aws_security_group" "interface_endpoint" {
```

Comment:
Could we use the tamr security group module instead like we are for the other security groups in this example? Or is there some limitation of the module?

Reply:
Fixed.